src/components/Autocomplete/index.tsx

```diff
@@ -181,7 +181,16 @@ class AutocompleteInt<T> extends React.Component<AutocompleteProps<T> & FormCont
       this._element.focus();
     }
 
-    suggestionSelected ? this.hide() : this.show();
+    const hide = () => {
+      this.hide();
+      setTimeout(() => {
+        if (this.state.open) {
+          this.hide();
+        }
+      }, 10);
```

Comment:
why `10`? Can you try to omit the second param? I guess it can also make it work.